src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -803,7 +803,7 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "Home_Risk_Last_Activate_Item_Unknown_Title" = "Unbekanntes Risiko";
 
-"Home_Risk_Last_Activate_Item_Low_Title" = "Niedriges Risiko";
+"Home_Risk_Last_Activate_Item_Low_Title" = "Geringes Risiko";
```

Comment:
The term "niedriges Risiko" is used in several places. For consistency reasons, this cannot be changed.